### ISSUE TITLE
improve the ui

### ISSUE THREAD

**Issue by Arman-ali-khan-786** [Arman-ali-khan-786/TexttoSpeech#3]
make the UI appealing ..

**Comment by Pdfulkar** [comment]
On it.

**Comment by Arman-ali-khan-786** [comment]
@Pdfulkar  great to see your energy go ahead!

**Comment by Pdfulkar** [comment]
[TextToSpeech.pdf](https://github.com/Arman-ali-khan-786/TexttoSpeech/files/9393648/TextToSpeech.pdf)
# 
Should I make a design like the above please suggest any changes.

**Comment by Pdfulkar** [comment]
I don't think the scroll bar other than speed is useful.
I think we should remove it.

**Comment by Arman-ali-khan-786** [comment]
> [TextToSpeech.pdf](https://github.com/Arman-ali-khan-786/TexttoSpeech/files/9393648/TextToSpeech.pdf)
> 
> Should I make a design like the above please suggest any changes.

No  theme should be dark  plus you can add an extra feature like file upload and then it will read that file's text ,for that create another issue and work on it

**Comment by Arman-ali-khan-786** [comment]
> I don't think the scroll bar other than speed is useful. I think we should remove it.

no let it be there ,you can add animations if possible on it or design it better

**Comment by Pdfulkar** [comment]
Ok I look into it

**Comment by Arman-ali-khan-786** [comment]
@Pdfulkar are you into it??

**Comment by Pdfulkar** [comment]
Yes it will take some time because of ganesh festival sorry for late i will give you prototype and ask for changes in two days.

**Comment by Pdfulkar** [comment]
https://www.figma.com/proto/hmR3fEm1q3XSNV80k3ylTP/texttospeechsite?node-id=1%3A2&scaling=scale-down&page-id=0%3A1
I created this layout.
If any suggestions are there tell me.
also, I am attaching the pdf file of the same.
[texttospeechsite.pdf](https://github.com/Arman-ali-khan-786/TexttoSpeech/files/9460545/texttospeechsite.pdf)

**Comment by Pdfulkar** [comment]
> you

Dark theme means please explain.

**Comment by Pdfulkar** [comment]
> > I don't think the scroll bar other than speed is useful. I think we should remove it.
> 
> no let it be there ,you can add animations if possible on it or design it better
ok

**Comment by Pdfulkar** [comment]
> > [TextToSpeech.pdf](https://github.com/Arman-ali-khan-786/TexttoSpeech/files/9393648/TextToSpeech.pdf)
> > Should I make a design like the above please suggest any changes.
> 
> No theme should be dark plus you can add an extra feature like file upload and then it will read that file's text ,for that create another issue and work on it

explain dark theme as in the background or in the text bar as I have done in the new design.

**Comment by Arman-ali-khan-786** [comment]
> > > [TextToSpeech.pdf](https://github.com/Arman-ali-khan-786/TexttoSpeech/files/9393648/TextToSpeech.pdf)
> > > Should I make a design like the above please suggest any changes.
> > 
> > 
> > No theme should be dark plus you can add an extra feature like file upload and then it will read that file's text ,for that create another issue and work on it
> 
> explain dark theme as in the background or in the text bar as I have done in the new design.

dark theme in the background , you can use standard dark colors and make the size of the control buttons smaller ok ..
rest seems ok to me

**Comment by rik-chatterjee** [comment]
Hi can I work on this issue, if it is not completed yet ?

**Comment by Pdfulkar** [comment]
Yah sure

**Comment by Arman-ali-khan-786** [comment]
> Hi can I work on this issue, if it is not completed yet ?

ok assigned

**Comment by Arman-ali-khan-786** [comment]
@rik-chatterjee any updates??

**Comment by Abhijit-codder** [comment]
i can help ig

**Comment by guranshdeol** [comment]
HI @Arman-ali-khan-786 , i am interested in improving the Ui to make it look more aesthetic .Please assign this to me and accept my upcoming PR's/
